config.json

```diff
@@ -0,0 +1,550 @@
+﻿{
```

Comment:
The size of this file is scary and we will need to iterate on this more overtime. However all the read in the individual scripts makes me very happy :)